src/core_modules/capture-core/components/ListView/index.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+export { default as EventList } from './ListWrapper/EventsListWrapper.container';
+export { actionTypes as paginationActionTypes } from './Pagination';
+export { actionTypes } from './listView.actions';
+
+export { dateFilterTypes, assigneeFilterModes } from '../FiltersForTypes';
```

Comment:
hey! Could you take a moment and explain the need to export those constants and the `type`s bellow from the index file of the `ListView` component?

Reply:
Yeah, The idea is that the `WorkingLists` uses the `ListView` components and is not aware of the existence of `FilterForTypes`. If the `WorkingLists` needs access to something from `FilterForTypes` it needs to do this through `ListView`. This will make more sense later because you will see that I will define an interface (props that are passed into the `ListView` main component from the `WorkingLists`) where filters are part of this interface. These filters are defined in the `FilterForTypes`because they do not change in the `ListView` context. You might want to know and use these types from the `WorkingLists`, but what you care about is what the `ListView` accepts, not the underlying `FilterForTypes`.